scripts/build-pktio-dpdk

```diff
@@ -28,7 +28,7 @@ sed -ri 's,(CONFIG_RTE_LIBRTE_PMD_PCAP=).*,\1y,' .config
 popd
 
 #Build DPDK
-make build O=${TARGET} EXTRA_CFLAGS="-fPIC"
+make -j $(nproc) build O=${TARGET} EXTRA_CFLAGS="-fPIC"
```

Comment:
This script is not used during ODP Travis builds.

Reply:
It's not used anymore. But some dependency/readme file refer to it.